ProcessHacker/proctree.c

```diff
@@ -1211,6 +1211,133 @@ static VOID PhpUpdateProcessNodeAppId(
     }
 }
 
+static ULONG PhpGet_gfDPIAwareOffset32(
+    /* user32VBase is the base the offset in the movzx is relative to,
+     * either the current load address if relocations are performed, or
+     * the default base if not. */
+    _In_ ULONG user32VBase,
+    _In_ PBYTE isProcessDPIAware32)
+{
+    // 0FB605xxxxxxxx    movzx eax, gfDPIAware
+    // C3                ret
+    __try
+    {
+        if (isProcessDPIAware32[0] == 0x0F && isProcessDPIAware32[1] == 0xB6 &&
```

Comment:
This is a maintenance nightmare.

Reply:
Could you clarify?

Reply:
We don't use these type of workarounds anywhere else in the codebase because that code will break every other time Microsoft patches something and since most other contributors would not know how to fix that code it would just be some poor soul (me) having to fix it for months/years into the future.